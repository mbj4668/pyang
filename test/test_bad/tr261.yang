module tr261 {
  namespace urn:tr261;
  prefix x;

  grouping a {
    leaf b {
      type uint8;
      mandatory true;
    }
  }

  container x {
    uses a {
      refine b {
        default "0"; // error; mandatory and default
      }
    }
  }

  container y {
    uses a {
      refine b {
        default "0";
        mandatory false;
      }
    }
  }

}
