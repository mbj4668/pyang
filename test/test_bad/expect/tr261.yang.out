tr261.yang:13 (at tr261.yang:6): error: DEFAULT_AND_MANDATORY
